src/neighborsearch/kball_searcher.jl

```diff
@@ -0,0 +1,52 @@
+# ------------------------------------------------------------------
+# Licensed under the ISC License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+KBallSearcher(object, neighborhood)
```

Comment:
Could you please prepend 4 whitespaces to this line? This way the Julia help prompt will highlight the function signature.